source/index.html.md.erb

```diff
@@ -8752,32 +8862,21 @@ List<Row> updatedRows = smartsheet.sheetResources().rowResources().updateRows(
 ```
 
 ```python
-# Specify updated cell values for one row
-row_a = smartsheet.Sheets.get_row(sheetId, 6572427401553796)
-
-# use row get_column(columnId) convenience method to pluck the 
-# cell with a known columnId out of the row
-cell_a = row_a.get_column(7518312134403972)
-cell_a.value = 'new value'
-cell_b = row_a.get_column(1888812600190852)
-cell_b.value = 'A'
-
-# use row set_column(id, Cell) convenience method
-# to replace the cell in-place
-row_a.set_column(cell_a.column_id, cell_a)
-row_a.set_column(cell_b.column_id, cell_b)
+# Build new cell value
+new_cell_spec = ss_client.models.Cell()
```

Comment:
This is a legitimate `cell` object. So I'd call it `new_cell`. I prefer "_spec" for the helper objects that provide info, but are a different type. Same for `new_row_spec` below. (P2)